docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  task_db:
+    image: postgres
+  task:
+    build: .
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8001"
```

Comment:
Do you mean for https://github.com/cognoma/core-service/issues/39 ? I think we should use an application server here as well, but docker compose is used for the development environment, not production.